internal/connect/command_custom_plugin_create.go

```diff
@@ -39,7 +39,7 @@ func (c *customPluginCommand) newCreateCommand() *cobra.Command {
 	cmd.Flags().String("plugin-file", "", "ZIP/JAR custom plugin file.")
 	cmd.Flags().String("connector-class", "", "Connector class of custom plugin.")
 	cmd.Flags().String("connector-type", "", "Connector type of custom plugin.")
-	cmd.Flags().String("cloud", "", "Cloud provider of custom plugin.")
+	cmd.Flags().String("cloud", "", "Cloud provider of custom plugin. Supported value [\"aws\",\"azure\"] (default \"aws\").")
```

Comment:
Use `pcmd.AddCloudFlag()` here, too?

Reply:
We do not support gcp right now, so might need to create a new function for this.
May I ask how can I get the completion suggestion while using this function?
I guess it is when you start typing the command in the terminal: `connect custom-plugin list --cloud` and hit the TAB key. it will show the suggestion. 
I make build use the new cli to test,  but not get the sugguestion.

Reply:
For create cmd, we want to let user know we have aws as the default value.
For list cmd, the default value should be "".
Would you think we need to have two autocompletion funcs?

Reply:
> I guess it is when you start typing the command in the terminal and hit the TAB key

Yep, that's correct. Make sure you follow the steps in `confluent completion -h` too.

> Would you think we need to have two autocompletion funcs?

Sounds like it, since they both autocomplete a different set of values.

Reply:
Also- please add an integration test for the autocompletion function in the `test/` directory, since this autocompletion is non-trivial